### ISSUE TITLE
SoapClient is missing on 0.12.1

### ISSUE THREAD

**Issue by ricardo-comar** [visiblevc/wordpress-starter#66]
Hi,

First I wanna thank you so much !! I'm using this project with **WooCommerce** plugin so far, and it's helping us a lot to simplify the environment setup. You guys rock!!!

Today we updated our local docker image to latest (0.12.1), and we noticed our WP plugin stopped working. We just realized that the **PHP SoapClient** is missing on latest version, but it's present until 0.11.0. **WooCommerce** also makes that check on System Status page. 

I believe that's an important dependency to be present on all versions, please check if it's possible to be added on next version.

BR

**Comment by dsifford** [comment]
Hi @ricardo-comar 

Thanks for the report. I've added the fix for this to #61 (just awaiting @karellm's approval for the merge).

To enable it in your current build, open up a shell session in the wordpress service and run the following command.

```
apt-get update && apt-get install -y libxml2-dev && docker-php-ext-install soap
``` 

Let me know if that doesn't work for you. 👍

**Comment by ricardo-comar** [comment]
Hi @dsifford 

I tested that snippet and it's working as expected. Thanks for the quick reply ! ;)

But for my team I fixed the version to 0.11.0 on docker-compose, it has all the features we need for now.

BR
